### DISCUSSION TITLE
Issues with IEA-15-240-RWT-UMaineSemi  Simulation

### DISCUSSION THREAD

**Discussion by Uisliu** [OpenFAST/openfast#2018] [Q&A]
Hi all,
I am looking for some assistance regarding simulations that I am trying to conduct for the IEA-15-240-RWT-UMaineSemi model So far I have;
Installed Openfast (Precompiled binaries)
Installed the IEA-15-240-RWT-UMaineSemi master file
Installed ROSCO (Precompiled along with the libdiscon.dll file)
When running the simulation Through OpenFAST I got the following errors;
![image](https://github.com/OpenFAST/openfast/assets/71447919/ecfc816f-0654-4646-8541-1d13acfbf720)
 
These errors seem to suggest that I am missing files of some sort. Could this be a problem with versions or am I missing a program altogether? 
Looking forward to some input : )

**Comment by jjonkman** [comment]
I would suggest posting your question on the ROSCO GitHub site: https://github.com/NREL/ROSCO/discussions.
